src/shared/api/CancerHotspotsAPI.ts

```diff
@@ -0,0 +1,1044 @@
+import * as request from "superagent";
+
+type CallbackHandler = (err: any, res ? : request.Response) => void;
+export type HotspotMutation = any;
+export type SingleResidueHotspotMutation = any;
+export type IntegerRange = any;
+export type TumorTypeComposition = any;
+export type Cluster = any;
+export type ClusteredHotspotMutation = any;
```

Comment:
Seems like auto generator misses the actual field names for these types.
@adufilie : Do you have any idea why the swagger code generator ended up declaring these types as `any`?